src/validateEmail.test.js

```diff
@@ -8,12 +8,76 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof validateEmail('')).toBe('boolean');
+  });
+
+  it(`should return 'true' for the email form
+  [name]@[domain].[top domain]`, () => {
+    expect(validateEmail('qas@mail.com')).toBe(true);
+  });
+
+  it(`should return 'true' for the  email
+  [name]@[domain].[top domain]`, () => {
+    expect(validateEmail('q@m.c')).toBe(true);
+  });
+
+  it(`should return 'false' for the email
+  with symbol `, () => {
```

Comment:
```suggestion
  it(`should return 'false' for the email with symbol `, () => {
```
which symbol? 
why in 2 lines? it requires only for the strings with more than 80 symbols